Gemfile.lock

```diff
@@ -135,8 +138,6 @@ GEM
     pundit (2.0.1)
       activesupport (>= 3.0.0)
     rack (2.0.7)
-    rack-protection (2.0.5)
```

Comment:
not sure if this should be being removed?

Reply:
Not sure why this was not updated, but if you view the Gemfile.lock file it has been re-added

Reply:
but that's only re sidekiq vs this top-line inclusion. would you rerun bundle install and see if this change is still part of the diff?